src/components/NavBarLink/NavBarLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { NavLink } from 'react-router-dom';
+import classnames from 'classnames';
+
+type Props = {
+  to: string;
+  name: string;
+};
+
+export const NavBarLink: React.FC<Props> = ({ to, name }) => (
+  <NavLink
+    className={({ isActive }) => (
+      classnames('navbar-item', {
+        isActive,
+      })
```

Comment:
in my opinion, this className don't work correct,

try { 'is-active': isActive}

Reply:
Thx missed this moment.